CONTRIBUTING.md

```diff
@@ -30,7 +30,7 @@ Run `/opt/bin/deis-debug-logs` on one of your CoreOS machines and provide the ou
 ## Getting Started
 
 - Fork the repository on GitHub
-- Read the README.md for build instructions
+- Read [the documentation](https://github.com/deis/deis/blob/master/docs/contributing/hacking.rst) for build instructions
```

Comment:
Instead of the ReStructuredText source, this should link to the hosted docs URL for the same thing: http://docs.deis.io/en/latest/contributing/hacking/